Library/Homebrew/test/test_bottle_versions.rb

```diff
@@ -75,4 +75,12 @@ def test_ngircd_style
     assert_version_detected '22',
       'ngircd-22.mavericks.bottle.tar.gz'
   end
+
+  def test_apparix_style
+    assert_version_detected '11-062',
+      'apparix-11-062.yosemite.bottle.tar.gz'
+
+    assert_version_detected '11-062',
+      'apparix-11-062..bottle.tar.gz'
```

Comment:
Because, such case will happen for all formulae when they get bottled in the first time. Related codes are in [here](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/pull.rb#L173-L174).